runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -1,20 +1,21 @@
-// Lists all cases that we want to measure.
+/// Lists all cases that we want to measure.
 pub mod cases;
-// Generates runtime fees from the measurements.
-pub mod runtime_fees_generator;
-// Generates external costs from the measurements.
-pub mod ext_costs_generator;
-// Runs a VM (Default: Wasmer) on the given contract and measures the time it takes to do a single operation.
 #[cfg(feature = "protocol_feature_evm")]
+/// Generates the runtime fees for the EVM.
 pub mod evm_estimator;
-pub mod vm_estimator;
-// Collects and processes stats. Prints them on display, plots them, writes them into a file.
+/// Generates external costs from the measurements.
+pub mod ext_costs_generator;
+/// Generates runtime fees from the measurements.
+pub mod runtime_fees_generator;
+/// Collects and processes stats. Prints them on display, plots them, writes them into a file.
 pub mod stats;
-// Encapsulates the runtime so that it can be run separately from the rest of the node.
+/// Encapsulates the runtime so that it can be run separately from the rest of the node.
 pub mod testbed;
-// Prepares transactions and feeds them to the testbed in batches. Performs the warm up, takes care
-// of nonces.
+/// Prepares transactions and feeds them to the testbed in batches. Performs the warm up, takes care
+/// of nonces.
 pub mod testbed_runners;
+/// Runs a VM (Default: Wasmer) on the given contract and measures the time it takes to do a single operation.
+pub mod vm_estimator;
```

Comment:
should these be `///`, Looks like they are comments instead of docstrings.

Reply:
Was something that `cargo fmt` picked up that isn't related to this work.